frontend/app/src/scenes/Home/scenes/Category/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import axios, { AxiosResponse } from 'axios';
+import { CategoryStateProps } from './interfaces';
+import { Translation, Category } from '../../interfaces';
+import { Col, Row, Table, Typography } from 'antd';
+import styles from './styles.css';
+import { handleApiError } from '../../../../services/util/errorHandler';
+const { Title } = Typography;
+
+const columns = [
+  {
+    title: 'id',
+    dataIndex: 'id',
+    key: 'id',
+  },
+  {
+    title: 'name',
+    dataIndex: 'translations',
+    key: 'name',
+    // eslint-disable-next-line react/display-name
+    render: (translations: Translation[]) => (
+      <Row>
+        {translations.map((translation) => {
+          return (
+            <Col key={translation.language}>
+              <p>
+                {translation.name} : {translation.language}
+              </p>
+            </Col>
+          );
+        })}
+      </Row>
```

Comment:
Shall we display only English translation? instead of iterating all. (let's use 0th index for now)

Reply:
Sure